components/Trade/Positions/AvailableTokens.tsx

```diff
@@ -103,7 +102,7 @@ export default function AvailableTokens({
 }
 
 function getColumns(
-  onBuyClick: HandleBuyClick,
+  _onBuyClick: HandleBuyClick,
```

Comment:
Is the added underscore for keeping linter happy or ..? :slightly_smiling_face:

Reply:
Yes. :D